modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -2,18 +2,29 @@
 //
 // <List assemblies here, each on a new line>
 
-// Base the file name and the ID on the module title. For example:
-// * file name: doing-procedure-a.adoc
-// * ID: [id="doing-procedure-a"]
-// * Title: = Doing procedure A
-
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
-[id="doing-one-procedure_{context}"]
-// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+////
+Base the file name and the ID on the module title. For example:
+* file name: proc-doing-procedure-a.adoc
+* ID: [id="doing-procedure-a_{context}"]
+* Title: = Doing procedure A
+////
+ The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
```

Comment:
```suggestion
The ID is an anchor that links to the module. Do not change it after you publish the module to avoid breaking existing links.
```

Reply:
OK. Updated the first part. But some groups do regularly change the IDs so we need to say avoid.